src/main/java/classes/Order.java

```diff
@@ -37,17 +31,8 @@ public class Order
     // An instance of Finder class
     private Finder finder = new Finder();
 
-    /*
-     * Constructor Name : Order()
-     * Parameters : title, price, pairing, discount
-     * Description : Alternate Constructor
-     */
-    // public Order(String title, double price, String pairing) 
```

Comment:
remove useless comments

Reply:
Sent Output

Reply:
The screenshot u sent for useless comments was already removed in the given pull request